src/templates/Gov/CreateProposal/index.tsx

```diff
@@ -0,0 +1,351 @@
+import { ethers } from 'ethers'
+import Tippy from '@tippyjs/react'
+import React, { FormEvent } from 'react'
+import { useConnectWallet, useSetChain } from '@web3-onboard/react'
+
+import { GovernorAlpha } from '@/constants/tokenAddresses'
+
+import useGov from '@/hooks/useGov'
+import { useAppDispatch } from '@/store/hooks'
+import { useVotingPower } from '@/hooks/query/useVotingPower'
+import { setModalAlertText } from '@/store/reducers/modalAlertText'
+
+import Button from '@/components/Button'
+import VotingPower from '@/components/VotingPower'
+import Breadcrumb from '@/components/Breadcrumb'
+import BreadcrumbItem from '@/components/Breadcrumb/BreadcrumbItem'
+import InputText from '@/components/Inputs/InputText'
+import MarkdownEditor from '@/components/MarkdownEditor'
+import TitleWithContador from './TitleWithContador'
+import ExternalLink from '@/components/ExternalLink'
+import AdvancedProposalOption from './AdvancedProposalOption'
+
+import * as S from './styles'
+
+const CreateProposal = () => {
+  const [proposalTitle, setProposalTitle] = React.useState('')
+  const [proposalDescription, setProposalDescription] = React.useState('')
+  const [proposalForumLink, setProposalForumLink] = React.useState('')
+  const [proposalFunction, setProposalFunction] = React.useState('')
+  const [proposalParamanters, setProposalParamanters] = React.useState('')
+  const [proposalAddress, setProposalAddress] = React.useState('')
+  const [isOpenAdvancedOptions, setIsOpenAdvancedOptions] =
+    React.useState(false)
+
+  const dispatch = useAppDispatch()
+  const { propose } = useGov(GovernorAlpha)
+  const [{ wallet, connecting }, connect] = useConnectWallet()
+  const [{ connectedChain }, setChain] = useSetChain()
+  const { data } = useVotingPower({ id: wallet?.accounts[0].address ?? '' })
+
+  const minimalVotingPower = Big(
+    data?.governances[0]?.totalVotingPower ?? '0'
+  ).mul('0.01')
+
+  function handleProposalTitle(event: React.ChangeEvent<HTMLInputElement>) {
+    setProposalTitle(event.target.value)
+  }
+
+  function handleProposalDescription({ text }: { text: string }) {
+    setProposalDescription(text)
+  }
+
+  function handleProposalFunctions(event: React.ChangeEvent<HTMLInputElement>) {
+    setProposalFunction(event.target.value)
+  }
+
+  function handleToggleChange() {
+    setIsOpenAdvancedOptions(prevState => !prevState)
+  }
+
+  function handleProposalAddress(event: React.ChangeEvent<HTMLInputElement>) {
+    setProposalAddress(event.target.value)
+  }
+
+  function handleProposalParamanters(
+    event: React.ChangeEvent<HTMLInputElement>
+  ) {
+    setProposalParamanters(event.target.value)
+  }
+
+  function extractFunctionsList(value: string) {
+    const regex = /(\w+\([^)]*\))/g
+
+    return value.replace(/\s/g, '').match(regex) ?? []
+  }
+
+  function extractParamentersTypeList(value: string) {
+    const regex = /\(([^)]*)\)/g
+
+    const paramentersList = [...value.matchAll(regex)]
```

Comment:
Typo on "parameters"

```suggestion
    const parametersList = [...value.matchAll(regex)]
```